### ISSUE TITLE
saveAs return empty function

### ISSUE THREAD

**Issue by azhaziqgoh** [eligrey/FileSaver.js#513]
Hi,

I add file saver in the head tag, as follow
```html
<script src="~/Scripts/FileSaver.min.js"></script>
```

And in my javascript just try to call following line of code
```js
saveAs(new Blob([buffer], { type: "application/octet-stream" }), "test.xlsx");
```
and nothing works. I try to console.log(saveAs) it return empty function. It happen for release2.0. For release1.3.8 this issue does not occuer

**Comment by Ruffio** [comment]
@azhaziqgoh Has your issue been resolved and can be closed?

**Comment by luanpotter** [comment]
@Ruffio my problem is that I'm using Ecma 6 import syntax instead of require, and that gives the error presented at 2.0.0

**Comment by cosme-benito** [comment]
Why is the ECMA6 import syntax not supported?

**Comment by bufke** [comment]
This problem appears fixed in 2.0.1 to me.

**Comment by azhaziqgoh** [comment]
Seems fixed already. Closing this
